src/views/ForgotPass.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Header from '../components/global/Header';
+import AuthHolder from '../components/global/AuthHolder';
+
+const ForgotPass = () => (
+  <div className="authentication">
+    <Header />
+    <AuthHolder />
+  </div>
+);
+
+export default ForgotPass;
```

Comment:
remove this file because there is a component with the same code in /views/authentication.js